tests/testthat/test-renderer4-ticks-grids-updates.R

```diff
@@ -0,0 +1,192 @@
+acontext("update axis ticks and grids")
+
+# Plots with axis updates
+mtcars$cyl <- as.factor(mtcars$cyl)
+
+no_updates <- ggplot()+geom_point(aes(mpg, disp, 
+                                      colour=cyl), 
+                                  data = mtcars)
+
+update_x <- no_updates+
+  theme_animint(update_axes=c("x"))
+update_y <- no_updates+
+  theme_animint(update_axes=c("y"))
+update_xy <- no_updates+
+  theme_animint(update_axes=c("x","y"))
+
+viz <- (list(g=no_updates, 
+            g1=update_x, 
+            g2=update_y, 
+            g3=update_xy))
```

Comment:
same here, more informative names would be nice. any reason that these tests are in a separate file? If the viz list is exactly the same I would suggest combining these two test files.